conda_smithy/ci_register.py

```diff
@@ -69,7 +69,7 @@ def travis_headers():
 
 
 def add_token_to_circle(user, project):
-    url_template = ('https://circleci.com/api/v1/project/{user}/{project}/envvar?'
+    url_template = ('https://circleci.com/api/v1.1/project/{user}/{project}/envvar?'
```

Comment:
The v1.1 API requires we specify the VCS-type. So please add `github` after `project`.